flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/mysql/MySqlDialect.java

```diff
@@ -126,4 +126,14 @@ public Set<LogicalTypeRoot> supportedTypes() {
                 LogicalTypeRoot.TIME_WITHOUT_TIME_ZONE,
                 LogicalTypeRoot.TIMESTAMP_WITHOUT_TIME_ZONE);
     }
+
+    @Override
+    public String appendUrlSuffix(String url) {
+        String suffix = "rewriteBatchedStatements=true";
+        if (url.contains("?")) {
+            return url + "&" + suffix;
```

Comment:
We should user's url `rewriteBatchedStatements=false` if the user configured url contains `rewriteBatchedStatements=false` instead of use the default property.